aws-lambda-java-log4j2/pom.xml

```diff
@@ -61,7 +61,32 @@
             <version>${log4j.version}</version>
         </dependency>
     </dependencies>
-
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.moditect</groupId>
+        <artifactId>moditect-maven-plugin</artifactId>
+        <version>1.0.0.Final</version>
```

Comment:
Latest version seems to be 1.2.2, could you please update it?
https://mvnrepository.com/artifact/org.moditect/moditect-maven-plugin/1.2.2.Final